calcom/cal.com
# feature/app wipe my cal

## What does this PR do?
The highly requested Wipe my Cal APP


## Type of change


## How should this be tested?

1. Install Wipe my Cal from app store
2. Go to /bookings and hit the button Wipe Today
3. Confirm rescheduling
4. See the magic

## Checklist

- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
